packages/matrix-identity-server/src/db/index.test.ts

```diff
@@ -31,6 +31,9 @@ describe('Id Server DB', () => {
 
   afterAll(() => {
     logger.close()
+    // if (fs.existsSync('./testdb.db')) {
+    //   fs.unlinkSync('./testdb.db')
+    // }
```

Comment:
Why drop this ?

Reply:
I thought it would be redundant with the AfterEach juste above that has : 
afterEach(() => {
    process.env.TEST_PG === 'yes' || fs.unlinkSync('./testdb.db')
  })

Reply:
OK